### ISSUE TITLE
SOW Page: Make SOW Page

### ISSUE THREAD

**Issue by fourthletter** [OpenTechFund/apply-app#57]
Compliance/Contracting fields on project 

- [ ] If a SOW form exists there is the ability to edit this 
- [ ] SOW form opens in a new page 
- [ ] SOW form has the following fields: 
project title 
personal/organisational name
email
phone 
address
project value 
proposed start 
proposed end 

 - [ ] From the application form page there is the ability to save and return to the project page 
- [ ] SOW page information updated based on content of project approval form (PAF)
- [ ] QA is complete

**Comment by Erioldoesdesign** [comment]
Design a pdf or 'output' that is easy to read

**Comment by Erioldoesdesign** [comment]
@frjo Di's github is @fourthletter we're going to keep pinging this poor person that isn't part of our project

**Comment by Erioldoesdesign** [comment]
I made this same mistake don't worry!

**Comment by ei8fdb** [comment]
> Is not all this information added in the PAF form? So all that is needed is a SOW view (page) plus PDF export.

@frjo Yes it is. I am going to propose a rewrite of this issue description. I'm putting together a first design for the SOW pdf file at the moment. Then we can discuss how the SOW gets exported.

**Comment by ei8fdb** [comment]
I've put together this initial design for the Scope of Work PDF printout. 

![Design option for Scope of Work PDF output](https://share.balsamiq.com/c/oowAKSKVe8TdPcRyo45Ps4.png)

I'd like a review from @OpenTechFund/devs @fourthletter @Erioldoesdesign.

**Comment by ei8fdb** [comment]
@fourthletter @frjo 

I'd like to propose a rewrite of this issue along the following lines: 

## User story

As OTF user (who? General Council?), I need Hypha to allow me to export “Scopes of Work” for projects we fund so that the US Government auditor can audit our project activities.

## Summary

OTF are required to provide auditors with a “bundle” of information about projects they fund.

Part of this bundle is a document/file called the “Scope of Work”. This Scope of Work is a subset of the information contained in the Project Approval Form.

## Possible workflows
There's a options available:

1. The minimum process could be something like:
- OTF prepare for audit
- a user (who?) goes to Hypha identifies the projects
- a user asks Fredrik for these projects to be exported
- Fredrik does the export, which creates a number of PDF files
- the user downloads/receives an email with pdfs
- the use then give these information bundles to the auditor

2. A more self-service process could be:
- OTF prepare for audit
- a user (who?) identify the project(s) they need to provide the information bundles for
- a user (who?) goes to Hypha highlights/selects the projects
- the user then "exports" the information bundle (the scope of work, the contract, what else?) and downloads the pdf files
- they then give these information bundles to the auditor

## What is required?

1. Minimum process
- design: the Scope of Work output file
- technical: @frjo to figure out how the pdfs get exported

2. Self-service process
- design: the process so the user can select the projects in Hypha and export the Scopes of Work
- design: the Scope of Work output file
- technical: @frjo and devteam figure out how the pdfs get exported

**Comment by Erioldoesdesign** [comment]
Couple of quick review comments:

1. In the issue description it says this is a 'legal document' does that mean there needs to be any legally identifying information on the pdf beyond the 'letter header' type of design you've added Bernard? e.g. An OTF staff members name, legal entity number/reg, legal physical address etc. I'm not super familiar with legally binding documents but if this is one, we should ensure it's air tight so to speak.

2. Similar to the first comment, if this is a legal document in .pdf format do we need to ensure that Hypha 'spits out' a non-editable .pdf? as in when you open it with acrobat it won't let you edit it? I've been on the receiving end of .pdf files like this but never 'set up' one. I think it's a 'read only' .pdf. https://community.adobe.com/t5/acrobat-sdk-discussions/how-to-create-a-read-only-pdf/m-p/3889018

**Comment by fourthletter** [comment]
On May 4 OTF & SimSec met and decided to pause the SOW template creation until OTF provides SimSec with a finalized version of the SOW.
